README.md

```diff
@@ -28,46 +28,88 @@ You are encouraged to provide a short document describing any assumptions and de
 
 ### Acceptance criteria
 1. The SLMM never goes outside of the dimensions of the garden as supplied during startup
-2. The output of the SLMM after it finished each action must be in the format: “{Time} – {Action taken place} – {current location of SLMM}”. This should only be written after the action has finished taking place.
+1. The output of the SLMM after it finished each action must be in the format: “{Time} – {Action taken place} – {current location of SLMM}”. This should only be written after the action has finished taking place.
 
 2 – Exercise for senior server-side positions
 -------------------------------------
-Please read the exercise for juniors first for some background context, but please do not implement it. After you have read the above, consider the below changes to the above requirements:
+Assume we are interested in creating a smart lawn mowing machine called SLMM (smart lawn mowing machine). Your task is to create the software that will run inside the SLMM itself as a server and a client that will be used by human operators to mow the lawn. This version of SLMM starts mowing immediately when deployed, and never stops. You also don't need to rotate it at all, and it only supports the following commands:
+
+1. Move one position up
+1. Move one positions down
+1. Move one position to the left
+1. Move one position to the right
+1. Set start location and lawn size (width and height)
+
+All move actions take one second to perform. Please emulate this using Thread.Sleep or Task.Wait in your code.
+
+### Deliverables
+Provide one application that represents the server application exposed through a Restfull API (you are free to use any web framework of your choice) which emulates the work of SLMM and one console application that accepts command input from console and communicates it to the SLMM server application. You should provide access to an online repository that hosts all code for all applications, libraries and test projects. The readme should have all necessary information on how to build, run and test the solution.
```

Comment:
I wanted to leave hosting to the implementor to make it easier.  As for testing, choice is really something we want to allow for evaluation. Comments?